__tests__/helpers.ts

```diff
@@ -89,6 +89,7 @@ export function makeActionSpies(shadow = false): ActionSpies {
 }
 
 function makePgClientMock() {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Please just disable this lint rule globally.

Reply:
Fixed!